SECURITY.md

```diff
@@ -64,3 +64,8 @@ findings (unless you prefer to stay anonymous, of course).
 Please report vulnerabilities by e-mail to the following address: 
 
 * cncf-argo-security@lists.cncf.io
+
+## Securing your Argo CD Instance
+
+See the [operator manual security page](docs/operator-manual/security.md) for 
+additional security information.
```

Comment:
Can we rephrase this a little bit please, e.g. to

> See the [operator manual security page](docs/operator-manual/security.md) for additional information about Argo CD's security features and how to make your Argo CD production ready.

Reply:
Yep, that's much better!